server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -104,29 +112,46 @@ public static void completeDefaults(Props props) {
 
   public static Properties defaults() {
     Properties defaults = new Properties();
-    defaults.put(ProcessProperties.SEARCH_CLUSTER_NAME, "sonarqube");
-    defaults.put(ProcessProperties.SEARCH_HOST, "127.0.0.1");
-    defaults.put(ProcessProperties.SEARCH_JAVA_OPTS, "-Xmx1G -Xms256m -Xss256k -Djna.nosys=true " +
+    defaults.put(SEARCH_CLUSTER_NAME, "sonarqube");
+    defaults.put(SEARCH_HOST, "127.0.0.1");
+    defaults.put(SEARCH_JAVA_OPTS, "-Xmx1G -Xms256m -Xss256k -Djna.nosys=true " +
       "-XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=75 -XX:+UseCMSInitiatingOccupancyOnly " +
       "-XX:+HeapDumpOnOutOfMemoryError");
-    defaults.put(ProcessProperties.SEARCH_JAVA_ADDITIONAL_OPTS, "");
-
-    defaults.put(ProcessProperties.WEB_JAVA_OPTS, "-Xmx512m -Xms128m -XX:+HeapDumpOnOutOfMemoryError");
-    defaults.put(ProcessProperties.WEB_JAVA_ADDITIONAL_OPTS, "");
-    defaults.put(ProcessProperties.CE_JAVA_OPTS, "-Xmx512m -Xms128m -XX:+HeapDumpOnOutOfMemoryError");
-    defaults.put(ProcessProperties.CE_JAVA_ADDITIONAL_OPTS, "");
-    defaults.put(ProcessProperties.JDBC_MAX_ACTIVE, "60");
-    defaults.put(ProcessProperties.JDBC_MAX_IDLE, "5");
-    defaults.put(ProcessProperties.JDBC_MIN_IDLE, "2");
-    defaults.put(ProcessProperties.JDBC_MAX_WAIT, "5000");
-    defaults.put(ProcessProperties.JDBC_MIN_EVICTABLE_IDLE_TIME_MILLIS, "600000");
-    defaults.put(ProcessProperties.JDBC_TIME_BETWEEN_EVICTION_RUNS_MILLIS, "30000");
+    defaults.put(SEARCH_JAVA_ADDITIONAL_OPTS, "");
+
+    defaults.put(PATH_DATA, "data");
+    defaults.put(PATH_LOGS, "logs");
+    defaults.put(PATH_TEMP, "temp");
+    defaults.put(PATH_WEB, "web");
+
+    defaults.put(WEB_JAVA_OPTS, "-Xmx512m -Xms128m -XX:+HeapDumpOnOutOfMemoryError");
+    defaults.put(WEB_JAVA_ADDITIONAL_OPTS, "");
+    defaults.put(CE_JAVA_OPTS, "-Xmx512m -Xms128m -XX:+HeapDumpOnOutOfMemoryError");
+    defaults.put(CE_JAVA_ADDITIONAL_OPTS, "");
+    defaults.put(JDBC_MAX_ACTIVE, "60");
+    defaults.put(JDBC_MAX_IDLE, "5");
+    defaults.put(JDBC_MIN_IDLE, "2");
+    defaults.put(JDBC_MAX_WAIT, "5000");
+    defaults.put(JDBC_MIN_EVICTABLE_IDLE_TIME_MILLIS, "600000");
+    defaults.put(JDBC_TIME_BETWEEN_EVICTION_RUNS_MILLIS, "30000");
+
+    defaults.put(CLUSTER_ENABLED, "false");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "false" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)